gpt_lw/gpt.py

```diff
@@ -1,17 +1,29 @@
+from dataclasses import dataclass
```

Comment:
maybe lets standardize file naming? can this be "model_zdc.py" and then once we decide which one is better we can just delete the other and change the one we use to "model.py"?

Reply:
Sure, good idea